aten/src/ATen/native/Normalization.cpp

```diff
@@ -637,8 +641,8 @@ std::tuple<Tensor, Tensor, Tensor> batch_norm_cpu(const Tensor& self, const c10:
   c10::MaybeOwned<Tensor> weight_maybe_owned = at::borrow_from_optional_tensor(weight_opt);
   const Tensor& weight = *weight_maybe_owned;
   const Tensor& bias = c10::value_or_else(bias_opt, [] {return Tensor();});
```

Comment:
note: I think all of the wrapper removals were supposed to be changed to use `MaybeOwned` in #56648, but only the first line in each group was changed